### PR TITLE
Wrote an initial sphinx documentation page

### PR DESCRIPTION
**Pull request by brownan**
Here's my initial documentation writeup. I'm new to enscons and SCons in general, so I would appreciate a look over it for accuracy and such. Thanks!

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,17 +1,486 @@
...
+        *This should be the tag used for architecture-dependent wheels unless you specifically
+        intend to provide cross-Python compatibility with other ABIs  and/or platforms.*
+
+    .. py:function:: enscons.get_universal_tag()
+
+        Returns "py2.py3-none-any" for a pure python distribution compatible with both Python 2
+        and 3.
+
+        You probably don't want to use this unless you're still supporting and testing on
+        Python 2. Use ``py3-none-any`` for pure Python distributions supporting only Python 3.
```
Comment (dholth) [inline docs/index.rst]: maybe we need a constant for the common pure-python tag

File: pyproject.toml [hunk 2]
```diff
@@ -28,7 +28,7 @@ url = "https://github.com/dholth/enscons"
 
 [project.optional-dependencies]
 cli = ["click"]
-doc = ["sphinx"]
+doc = ["sphinx", "sphinx-rtd-theme"]
```
Comment (dholth) [inline pyproject.toml]: I like the Furo theme

**Comment by dholth** [comment]
This is great!

**Review by dholth — APPROVED** [review]
Thanks, this is great!

**Comment by brownan** [comment]
Cool! Thanks for looking over it, glad it's helpful.

I've been working on some new ideas. I'll open an issue soon for discussion.

**Pull request merged**